config/environments/production.rb

```diff
@@ -60,7 +60,7 @@
   config.logger = ActiveSupport::TaggedLogging.logger($stdout, formatter: config.log_formatter)
 
   # Change to "debug" to log everything (including potentially personally-identifiable information!)
-  config.log_level = ENV.fetch('RAILS_LOG_LEVEL', 'info')
+  config.log_level = ENV.fetch('RAILS_LOG_LEVEL', 'info')&.to_sym
```

Comment:
This is probably fine and will fix the comparison, but two thoughts:

- The template this comes from uses a string - https://github.com/rails/rails/blob/v8.0.2.1/railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt#L45 - which makes me think this value is meant to be set from this config value, but then maybe not checked/introspected that way? I think the logger level setters are pretty resilient and will assign the right internal integer log level whether they get a string/symbol/int
- Could try something like `Rails.logger.debug?` in that httplog initializer instead of the string/symbol thing -- in case something else other than this `config.log_level` manipulates the log level?

Reply:
I think this would be a better alternative indeed!

Reply:
Do you want to open such a PR to replace this one?